### PR TITLE
Psfrag

### PR DESCRIPTION
**Pull request by brucemiller**
This PR arranges for empty `ltx:picture` elements (that have a tex attribute) to generate an image via LaTeX, whether or not the `--pictureimages` commandline switch is used. (If they are non-empty, the content determines the conversion, depending on (possibly) other post-processors).

This technique allows us to improve the `psfrag` bindng by collecting `\psfrag` calls to be used with an `\includegraphics` (or `\epsfbox`) and get LaTeX to combine them into a single image, rather than discard them.

It also allows us to write a binding for the `overpic` package.

Along the way, a few other random bugs were fixed.

### CODE REVIEW

**Comment by dginev** [comment]
Fascinating! The code reads very nice to me, the upgrades are quite sensible. I'll do a couple of tests and write back.

**Comment by dginev** [comment]
I drew a random test (for extra fairness), with `math-ph/0011028`. Conversion succeeds without any issues, and I see a good match between the HTML and PDF. 

So this looks good from all sides! I'll go nitpick the code some more, but I couldn't find any obvious bits to protest on my first read.

Details left foldable:
<details>

```tex
\begin{figure}[h]
\psfrag{a}{$\tau_{1,2}$}
\begin{center}
\includegraphics[width=10cm]{permutinglegs.eps}
\end{center}
\caption{The action of a transposition}
\label{transposition}
\end{figure}
```

HTML:
image


PDF:
image

</details>

**Comment by dginev** [comment]
I guess one could protest the sizing being mildly peculiar, but that's minor. The figure I mentioned above looks a bit small in the HTML, due to the narrow width/height in the style attribute.

Should we have both a `width` and `height` attribute for `` _and_ width and height set from within `style`? Maybe we can choose one over the other (at some point).

```html
<img id="S6.F4.pic1" class="ltx_picture ltx_centering" 
   style="width:284.0pt;height:59.3pt;vertical-align:-2px" src="pic/pic1.png"
   alt="\psfrag{a}{$\tau_{1,2}$}\includegraphics[width=284.52756pt]{permutinglegs.eps}" width="529" height="113">
```

**Review by dginev — APPROVED** [review]

File: lib/LaTeXML/Package/graphics.sty.ltxml [hunk 1]
```diff
@@ -232,19 +234,47 @@ DefConstructor('\graphicspath DirectoryList', sub {
 DefMacro('\includegraphics OptionalMatch:* [][] Semiverbatim',
   '\@includegraphics#1[#2][#3]{#4}');
 
+sub graphics_get_candidates {
+  my ($path, $kv) = @_;
+  $path = ToString($path); $path =~ s/^\s+//; $path =~ s/\s+$//;
+  $path =~ s/^("+)(.+)\g1$/$2/;                           # unwrap if in quotes
```
Comment (dginev) [inline lib/LaTeXML/Package/graphics.sty.ltxml:240]: One of these days we should have a `LaTeXML::Util::Pathname` helper for these path cleaning tricks.

**Comment by brucemiller** [comment]
Yeah, I was wondering if the "missizing" was due to my code, or the lack 
of your beautification. Having style and size is certainly awkward, 
probably a resize?

**Pull request merged**